gnmvidispine/vidispine_api.py

```diff
@@ -352,15 +357,17 @@ def chunked_upload_request(self,upload_io,total_size,chunk_size,
         self.logger.debug("Commencing upload from {0} in chunks of {1}".format(upload_io,chunk_size))
         self.logger.debug("uploading to {0} with account {1}".format(self.host,self.user))
 
+        size_to_send = int(str(total_size))
+
         for startbyte in range(0,total_size,chunk_size):
             headers = {
-                'size': chunk_size,
+                'size': size_to_send,
                 'index': startbyte
             }
             upload_io.seek(startbyte,my_seek_set)
             body_buffer = upload_io.read(chunk_size)
             self.raw_request(path,method=method,matrix=matrix,query=query_params,body=body_buffer,
-                             content_type=content_type,extra_headers=headers)
+                             content_type=content_type,rawData=True,extra_headers=headers)
```

Comment:
the default value for rawData should be False, as that is the previous behaviour.  rawData should ONLY be True if we're passing binary data through - otherwise the text should be processed

Reply:
Is not a chunked upload always going to be binary data?

Reply:
I have written in an if statement which controls the value of this based on the content_type.